docs/csharp/methods.md

```diff
@@ -223,8 +225,9 @@ If a method is passed an array as an argument and modifies the value of individu
 
 [!code-csharp[csSnippets.Methods#101](../../samples/snippets/csharp/concepts/methods/returnarray1.cs#101)]
 
- <a name="extension"></a>
- ## Extension methods ##
+<a name="extension"></a>
+
+## Extension methods ##
```

Comment:
do you want to review all the headings in this topic to make them atx style?

Reply:
Sure, I'll take a look later, but I think that is a separate MarkdownLint rule. I was just re-enabled the non-passing rules one at a time

Reply:
Done. I'll do a separate PR to clean these up after this lands